watertap/unit_models/uv_aop.py

```diff
@@ -234,43 +236,26 @@ def build(self):
 
         units_meta = self.config.property_package.get_metadata().get_derived_units
 
-        # TODO: update IDAES such that solvent and solute lists are automatically created on the parameter block
```

Comment:
When you say this, you are saying in the NDMA_prop_pack correct? This is how ion_DSPMDE_prop_pack does it which may be a useful reference.

Reply:
This comment is deprecated and I copied from other files, I should delete.